docs/installation/installing-with-npm.md

```diff
@@ -15,7 +15,14 @@ To use GOV.UK Frontend with NPM you must:
    you don’t already have one. You can create a default `package.json` file by
    running `npm init` from the root of your application.
 
-3. If you want to use the GOV.UK Frontend Nunjucks macros, install Nunjucks -
+3. Install either:
```

Comment:
Annoying pedant comment :nerd_face: :

I'm not a pro on when to use 'hyphens' vs endash vs emdash but should we be using a hyphen here?

Reply:
Well spotted! The wider world may well use a dash here (and I would have used a dash in previous roles at other orgs). But the GOV.UK style is to use a hyphen. Neither Stephen nor I have the history on why exactly...